ui/src/app/routing/app-routing.tsx

```diff
@@ -239,6 +239,7 @@ export const AppRoutingComponent: React.FunctionComponent<RoutingProps> =
 
     return (
       <React.Fragment>
+        <div id='outdated' /> {/* for outdated-browser-rework */}
```

Comment:
could you add more context about how this works?

Reply:
I can find out and add some comments